### ISSUE TITLE
i'm facing an issue when i run python server.py command

### ISSUE THREAD

**Issue by Shivesh72** [insidesherpa/JPMC-tech-task-1-PY3#163]
it is showing mising parenthesis actually the server file contains old sytax i think it does not uses parenthesis in print and i'm using python 3 so it is causing an error
error:
C:\Users\shivesh raj\JPMC-tech-task-1>python server.py
  File "server.py", line 217
    print 'HTTP server started on port 8080'
          ^
SyntaxError: Missing parentheses in call to 'print'. Did you mean print('HTTP server started on port 8080')?

**Comment by Shivesh72** [comment]
please help me soon to do it properly i want to do it

**Comment by Pranita-7** [comment]
I'm also facing same issue,please help!

**Comment by Shivesh72** [comment]
Which Python version are you using?

**Comment by Shivesh72** [comment]
If you are using python 3 then write the command Python datafeef/server3.py

**Comment by Pranita-7** [comment]
I m using python 3.7.4 and I tried datafeef but it is again giving me the error of Errno 2 no such directory or file

**Comment by Shivesh72** [comment]
Have you cloned the second file of jp Morgan py3 wala if yes then use this
python datafeed/server3.py sorry you were using the typing mistake it's
datafeed

**Comment by Pranita-7** [comment]
Yes I cloned the second directory and on cloning I got the output "HTTP server started on port 8080" but after this I am not getting anything and also on running the client side I am getting nothing so How shall I proceed further?

**Comment by subhanshu04** [comment]
i am also facing the same issue

**Comment by Someshsw1109** [comment]
> it is showing mising parenthesis actually the server file contains old sytax i think it does not uses parenthesis in print and i'm using python 3 so it is causing an error
> error:
> C:\Users\shivesh raj\JPMC-tech-task-1>python server.py
>   File "server.py", line 217
>     print 'HTTP server started on port 8080'
>           ^
> SyntaxError: Missing parentheses in call to 'print'. Did you mean print('HTTP server started on port 8080')?



#*
I think this error message means that you're attempting to run a program that uses the Python 2 print statement...  Try print("Write here what you want to print") like this..
